medusa/init/__init__.py

```diff
@@ -11,6 +11,10 @@
 import site
 import sys
 
+import six
```

Comment:
Remove this import
1. It's unused
2. It isn't available for import here because the vendored libs are still not in `sys.path`.

Reply:
done